CODEOWNERS

```diff
@@ -47,7 +47,7 @@ modules/sdk-coin-cspr/ @BitGo/coins
 modules/sdk-coin-dot/ @BitGo/coins
 modules/sdk-coin-eos/ @BitGo/coins
 modules/sdk-coin-hbar/ @BitGo/coins
-modules/sdk-coin-near/ @BitGo/coins
+modules/sdk-coin-near/ @BitGo/blockchain-integration
```

Comment:
you can remove this  https://github.com/BitGo/BitGoJS/pull/2828 will be conflict.